index.js

```diff
@@ -22,6 +35,28 @@ export class UpdateAPK {
       });
   };
 
+  // 1.0 vs 1.1 return -1
+  // 2.2 vs 2.1 return 1
+  // 2.2.1 vs 2.2.01 return 0
+  checkVersion = (ver1, ver2) => {
```

Comment:
Version comparison is non-trivial. We should not reimplement it, we should use https://www.npmjs.com/package/semver - can you re-do this using that library as a dependency?